components/constraints.js

```diff
@@ -12,8 +12,11 @@ class Constraints extends Component {
     if (!constraints) return <div />;
     return (
       <ul style={{ listStyle: 'none', paddingLeft: 0 }}>
-        {(constraints.get('default') || constraints.get('default') === 0) &&
-          <li>default value: {constraints.get('default')}</li>
+        {constraints.has('default') &&
+          (constraints.get('type') === 'boolean' ?
+            (constraints.get('default') ? 'default value: true' : 'default value: false') :
```

Comment:
Ah, so false prints nothing.  OK, so the important part though is you should not be testing for `"type": "boolean"` in the schema, but as the default value itself.  There is no requirement that a schema with `true` or `false` as a value has its `"type"` field set to `"boolean"`.